src/xrCore/_math.cpp

```diff
@@ -215,9 +215,49 @@ XRCORE_API u64 QPC() noexcept
     return _dest;
 }
 
+#if defined(XR_ARM)
+#include <sys/syscall.h>
+#include <linux/perf_event.h>
```

Comment:
Please, move includes upper, to line 51.

Reply:
Check last commit.